test/helpers/kafka.go

```diff
@@ -21,12 +19,13 @@ type kafkaReceiver struct {
 	topics []string
 }
 
-func (kr *kafkaReceiver) ApplicationLogs(timeToWait time.Duration) (logs, error) {
+func (kr *kafkaReceiver) ApplicationLogs(timeToWait time.Duration) (string, error) {
 	logs, err := kr.tc.consumedLogs(kr.app.Name, loggingv1.InputNameApplication)
 	if err != nil {
-		return nil, fmt.Errorf("Failed to read consumed application logs: %s", err)
+		return "", fmt.Errorf("Failed to read consumed application logs: %s", err)
 	}
-	return logs.ByIndex(ProjectIndexPrefix), nil
+	// FIXME(alanconway) return only app- logs here?
```

Comment:
Please elaborate why switching back to check string instead of the typed form for `logs` as provided by ` test/helpers/types.go`